db/migrate/20220419193414_content_settings_to_dsl_category.rb

```diff
@@ -0,0 +1,5 @@
+class ContentSettingsToDslCategory < ActiveRecord::Migration[6.0]
+  def up
+    Setting.where(category: 'Setting::Content').update_all(category: 'Setting')
+  end
```

Comment:
I modeled this after https://github.com/theforeman/foreman-tasks/pull/659 and others which seem to be irreversible, but don't have `down` methods that throw `ActiveRecord::IrreversibleMigration`. I guess it's okay?

Reply:
Yeah, the error means, that it is point of no return for users, we should think twice before introducing such a heavy caliber. Tho it would not hurt to have such thinks, because there are certainly some points of no return :)